lib/n3h-ipc/n3hMode.js

```diff
@@ -65,6 +67,11 @@ class N3hMode extends AsyncClass {
 
     this.$pushDestructor(async () => {
       if (this._ipc) {
+        this._ipcSend('terminated')
+        for (let id of this._ipc.keys()) {
+          log.t('TERMINATED - closing', id)
+          await this._ipc.close(id)
+        }
```

Comment:
should we move this close loop into the _ipc destructor?

Reply:
Tried it, and it actually breaks the 'full api' test.
Currently the connection-iface doesn't close on destroy and doing that seems implementation dependent.